src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -408,6 +410,7 @@ class AgentCompletions extends Disposable {
 							label: withSlash,
 							insertText: `${withSlash} `,
 							detail: c.description,
+							documentation: c.description,
```

Comment:
This will duplicate the description in the flyout

Reply:
That might be okay, given this is currently broken otherwise.

Reply:
Actually, it just no longer shows the flyout. I think we should just check if in screen reader mode here

![Screenshot 2023-10-23 at 1 51 48 PM](https://github.com/microsoft/vscode/assets/29464607/a92a5536-2c3c-4ad2-a27e-5c612e429592)